torchvision/csrc/cuda/ROIPool_cuda.cu

```diff
@@ -58,51 +56,51 @@ __global__ void RoIPoolFForward(const int nthreads, const T* bottom_data,
     T maxval = is_empty ? 0 : -FLT_MAX;
     // If nothing is pooled, argmax = -1 causes nothing to be backprop'd
     int maxidx = -1;
-    const T* offset_bottom_data =
-        bottom_data + (roi_batch_ind * channels + c) * height * width;
+    const T* offset_input =
+        input + (roi_batch_ind * channels + c) * height * width;
     for (int h = hstart; h < hend; ++h) {
       for (int w = wstart; w < wend; ++w) {
-        int bottom_index = h * width + w;
-        if (offset_bottom_data[bottom_index] > maxval) {
-          maxval = offset_bottom_data[bottom_index];
-          maxidx = bottom_index;
+        int input_index = h * width + w;
+        if (offset_input[input_index] > maxval) {
+          maxval = offset_input[input_index];
+          maxidx = input_index;
         }
       }
     }
-    top_data[index] = maxval;
+    output[index] = maxval;
     argmax_data[index] = maxidx;
   }
 }
 
 template <typename T>
-__global__ void RoIPoolFBackward(const int nthreads, const T* top_diff,
+__global__ void RoIPoolBackward(const int nthreads, const T* grad_output,
     const int* argmax_data, const int num_rois, const T spatial_scale,
     const int channels, const int height, const int width,
-    const int pooled_height, const int pooled_width, T* bottom_diff,
-    const T* bottom_rois) {
-  CUDA_1D_KERNEL_LOOP(index, nthreads) {
-    // (n, c, ph, pw) is an element in the pooled output
-    int pw = index % pooled_width;
-    int ph = (index / pooled_width) % pooled_height;
-    int c = (index / pooled_width / pooled_height) % channels;
-    int n = index / pooled_width / pooled_height / channels;
-
-    const T* offset_bottom_rois = bottom_rois + n * 5;
-    int roi_batch_ind = offset_bottom_rois[0];
-    int bottom_offset = (roi_batch_ind * channels + c) * height * width;
-    int top_offset    = (n * channels + c) * pooled_height * pooled_width;
-    const T* offset_top_diff = top_diff + top_offset;
-    T* offset_bottom_diff = bottom_diff + bottom_offset;
-    const int* offset_argmax_data = argmax_data + top_offset;
-
-    int argmax = offset_argmax_data[ph * pooled_width + pw];
-    if (argmax != -1) {
-      atomicAdd(
-          offset_bottom_diff + argmax,
-          static_cast<T>(offset_top_diff[ph * pooled_width + pw]));
-
+    const int pooled_height, const int pooled_width, 
+    T* grad_input, const T* rois, 
+    const int n_stride, const int c_stride,
+    const int h_stride, const int w_stride) {
+
+    CUDA_1D_KERNEL_LOOP(index, nthreads) {
```

Comment:
nit: for next time, try not to modify the indentation of the code, as it makes reviewing easier. Also, we are using 2 spaces for C++ files, so the previous indentation was ok. I'm merging this in anyway, but just wanted to let you know

Reply:
Understood, will keep that in mind.